Assets/Scripts/Entities/PhysicsProps/Wind.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using MLAPI;
+
+[AddComponentMenu("PhysicsProps/Wind")]
+public class Wind : PhysicsProp {
+    [Header("Constant forces")]
+    public bool isConstant;
+    public bool isLocal;
+    public Vector3 constantForceValue;
+    public Vector3 constantVelocityValue;
+
+    public Vector3 GetForceAtPosition(Vector3 position) {
+        Vector3 calculatedForce = Vector3.zero;
+        if (isConstant) calculatedForce = constantForceValue;

```

Comment:
Maybe put a comment in here for us later that says that only constant wind is supported